keycloak-iam/src/main/java/com/valtimo/keycloak/service/KeycloakUserManagementService.java

```diff
@@ -61,16 +58,16 @@ public class KeycloakUserManagementService implements UserManagementService {
 
     private final KeycloakService keycloakService;
     private final String clientName;
-    private final CacheManager cacheManager;
+    private final UserCache userCache;
 
     public KeycloakUserManagementService(
         KeycloakService keycloakService,
         String keycloakClientName,
-        CacheManager cacheManager
+        UserCache userCache
     ) {
         this.keycloakService = keycloakService;
         this.clientName = keycloakClientName;
-        this.cacheManager = cacheManager;
+        this.userCache = userCache;
```

Comment:
Caught between a rock and a hard place here I think, but now this is breaking too.